quirks/password-rules.json

```diff
@@ -41,6 +41,9 @@
     "americanexpress.com": {
         "password-rules": "minlength: 8; maxlength: 20; max-consecutive: 4; required: lower, upper; required: digit; allowed: [%&_?#=];"
     },
+    "ana.co.jp": {
+        "password-rules": "minlength: 8; maxlength: 16; required: digit; required: upper,lower;"
```

Comment:
What I got from the screenshot is that only alphanumeric characters are allowed, in which case shouldn't this use "allowed:" like the one below?

Reply:
Site requires a combination of alphabets and numbers (see image below). Attached it to the original PR comment
<img width="617" alt="Screenshot 2024-05-29 at 4 12 23 AM" src="https://github.com/apple/password-manager-resources/assets/88408806/f54542fd-d557-4f1a-b223-3f7904d8b642">

Reply:
Yeah, it requires passwords to have both a number and an alphabet. Passwords such as "abcdefghij" and "123456789" spit out the error message "Please enter your web password by combining alphabets and numbers". I believe the error message is kind of vague because it's translated from Japanese.